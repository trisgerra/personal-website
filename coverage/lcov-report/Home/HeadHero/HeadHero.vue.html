<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for Home/HeadHero/HeadHero.vue</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../prettify.css" />
    <link rel="stylesheet" href="../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../index.html">All files</a> / <a href="index.html">Home/HeadHero</a> HeadHero.vue
    </h1>
    <div class='clearfix'>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51
52
53
54
55
56
57
58
59
60
61
62
63
64
65
66
67
68
69
70
71
72
73
74
75
76
77
78
79
80
81
82
83
84
85
86
87
88
89
90
91
92
93
94
95
96
97
98
99
100
101
102
103
104
105
106
107
108
109
110
111
112
113
114
115
116
117
118
119
120
121
122
123
124
125
126
127
128
129
130
131
132
133
134
135
136
137
138</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">10x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">10x</span>
<span class="cline-any cline-yes">12x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">12x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">12x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">&lt;template&gt;
  &lt;div&gt;
    &lt;div class="container desktop is-hidden-touch"&gt;
      &lt;div class="absolute columns"&gt;
        &lt;div&gt;
          &lt;img class="profile" src="../../../assets/profile2.jpg" alt="profile image" /&gt;
        &lt;/div&gt;
        &lt;div class="column text-container"&gt;
          &lt;h2 class="title"&gt;{{title}}&lt;/h2&gt;
          &lt;h3 class="subtitle"&gt;{{subtitle}}&lt;/h3&gt;
          &lt;p class="description" v-html="description"&gt;&lt;/p&gt;
        &lt;/div&gt;
      &lt;/div&gt;
      &lt;nav class="columns"&gt;
        &lt;div class="column first"&gt;&lt;/div&gt;
        &lt;div class="column"&gt;&lt;/div&gt;
        &lt;div class="column"&gt;&lt;/div&gt;
        &lt;div class="column"&gt;&lt;/div&gt;
        &lt;div class="column"&gt;&lt;/div&gt;
        &lt;div class="column"&gt;&lt;/div&gt;
      &lt;/nav&gt;
    &lt;/div&gt;
    &lt;div class="container mobile is-hidden-desktop"&gt;
      &lt;div class="columns"&gt;
        &lt;div class="column is-12-mobile"&gt;
          &lt;img class="profile-mobile" src="../../../assets/profile2.jpg" alt="profile image" /&gt;
        &lt;/div&gt;
       &lt;div class="column text-container"&gt;
          &lt;h2 class="title"&gt;{{title}}&lt;/h2&gt;
          &lt;h3 class="subtitle"&gt;{{subtitle}}&lt;/h3&gt;
          &lt;p class="description" v-html="description"&gt;&lt;/p&gt;
        &lt;/div&gt;
      &lt;/div&gt;
    &lt;/div&gt;
  &lt;/div&gt;
&lt;/template&gt;
&nbsp;
&lt;script lang="ts"&gt;
import { Component, Vue } from 'vue-property-decorator';
&nbsp;
@Component({
  components: {},
})
export default class HeadHero extends Vue {
  title: string = 'Andrea DG &lt;website/&gt;';
&nbsp;
  subtitle: string = 'WHO AM I?';
&nbsp;
  description: string = "I'm &lt;b&gt;simply&lt;/b&gt; a Software developer with a passion for &lt;b&gt;new technologies&lt;/b&gt; and the &lt;b&gt;design&lt;/b&gt; ."
}
&lt;/script&gt;
&nbsp;
&lt;style lang="scss"&gt;
&nbsp;
.desktop {
  border-left: 3px solid #FF0000;
  position: relative;
  max-width: 1920px;
  border-bottom: 1px solid #dfdfdf;
&nbsp;
  .column {
    height: 40vw;
    border-right: 1px solid #dfdfdf;
  }
&nbsp;
  .text-container {
    margin-left: 30px;
  }
&nbsp;
  .description {
    font-size: 1.3vw;
    font-weight: 500;
    color: #3f3c3c;
    margin-left: 20px;
    margin-bottom: 0px;
  }
&nbsp;
  .first{
    border: 0px;
  }
  .columns {
    position: relative;
    width: 100%;
    margin: 0px;
  }
  .absolute {
    position: absolute;
    margin-left: 30px;
    margin-top: 10%;
  }
  .profile {
    border: 5px solid #151616;
    border-radius: 50%;
    width: 20vw;
    height: 20vw;
  }
&nbsp;
  .title {
    color: #3f3c3c;
    font-size: 4vw;
    font-weight: bold;
    margin-bottom: 40px;
  }
&nbsp;
  .subtitle {
      font-size: 2vw;
      font-weight: 500;
      color: #3f3c3c;
  }
}
&nbsp;
&nbsp;
.mobile {
  border-left: 3px solid #ff0000;
  padding:64px;
  text-align: center;
&nbsp;
  .profile-mobile {
    border: 5px solid #151616;
    border-radius: 50%;
  }
&nbsp;
  .title {
    color: #3f3c3c;
    font-size: 6vw;
    font-weight: bold;
  }
&nbsp;
  .subtitle {
      font-size: 4vw;
      font-weight: 500;
      color: #3f3c3c;
  }
}
&nbsp;
&nbsp;
&lt;/style&gt;
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Tue Apr 14 2020 12:35:07 GMT+0200 (Central European Summer Time)
</div>
</div>
<script src="../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../sorter.js"></script>
</body>
</html>
