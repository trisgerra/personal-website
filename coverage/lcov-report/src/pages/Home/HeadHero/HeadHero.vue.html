<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for src/pages/Home/HeadHero/HeadHero.vue</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../../index.html">All files</a> / <a href="index.html">src/pages/Home/HeadHero</a> HeadHero.vue
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>6/6</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>1/1</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>5/5</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51
52
53
54
55
56
57
58
59
60
61
62
63
64
65
66
67
68
69
70
71
72
73
74
75
76
77
78
79
80
81
82
83
84
85
86
87
88
89
90
91
92
93</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">11x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">11x</span>
<span class="cline-any cline-yes">12x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">12x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">12x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">&lt;template&gt;
  &lt;div class="container"&gt;
    &lt;div class="absolute columns"&gt;
      &lt;div&gt;
        &lt;img class="profile" src="../../../assets/profile.jpg" alt="profile image" /&gt;
      &lt;/div&gt;
      &lt;div class="column text-container"&gt;
        &lt;h2 class="title"&gt;{{title}}&lt;/h2&gt;
        &lt;h3 class="subtitle"&gt;{{subtitle}}&lt;/h3&gt;
        &lt;p class="description"&gt;{{description}}&lt;/p&gt;
      &lt;/div&gt;
    &lt;/div&gt;
    &lt;nav class="columns"&gt;
      &lt;div class="column first"&gt;&lt;/div&gt;
      &lt;div class="column"&gt;&lt;/div&gt;
      &lt;div class="column"&gt;&lt;/div&gt;
      &lt;div class="column"&gt;&lt;/div&gt;
      &lt;div class="column"&gt;&lt;/div&gt;
      &lt;div class="column"&gt;&lt;/div&gt;
    &lt;/nav&gt;
  &lt;/div&gt;
&lt;/template&gt;
&nbsp;
&lt;script lang="ts"&gt;
import { Component, Vue } from 'vue-property-decorator';
&nbsp;
@Component({
  components: {},
})
export default class HeadHero extends Vue {
  title: string = 'Welcome to my &lt;space/&gt;';
&nbsp;
  subtitle: string = 'WHO AM I?';
&nbsp;
  description: string = "I'm a enthusiast Software developer but first of all I consider myself as ethernal curious,&lt;/br&gt; that loves to learn, new things everyday."
}
&lt;/script&gt;
&nbsp;
&lt;style lang="scss" scoped&gt;
.text-container {
  margin-left: 30px;
}
.container {
  border-left: 3px solid #FF0000;
  position: relative;
  max-width: 1920px;
  border-bottom: 1px solid #dfdfdf;
}
.column {
  height: 600px;
  border-right: 1px solid #dfdfdf;
}
&nbsp;
.first{
  border: 0px;
}
.columns {
  position: relative;
  width: 100%;
  margin: 0px;
}
.absolute {
  position: absolute;
  margin-left: 30px;
  margin-top: 10%;
}
.profile {
  border: 5px solid #151616;
  border-radius: 50%;
  width: 300px;
  height: 300px;
}
&nbsp;
.title {
  color: #3f3c3c;
  font-size: 70px;
  font-weight: bold;
  margin-bottom: 40px;
}
&nbsp;
.subtitle {
    font-size: 22px;
    font-weight: 500;
    color: #3f3c3c;
    margin-left: 20px;
    margin-bottom: 0px;
}
&nbsp;
p {
    margin-left: 20px;
}
&lt;/style&gt;
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Sat Jul 13 2019 23:20:43 GMT+0200 (Central European Summer Time)
</div>
</div>
<script src="../../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../../sorter.js"></script>
</body>
</html>
