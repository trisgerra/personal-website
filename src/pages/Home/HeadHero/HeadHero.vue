<template>
  <div class="container">
    <div class="absolute columns">
      <div>
        <img class="profile" src="../../../assets/profile.jpg" alt="profile image" />
      </div>
      <div class="column text-container">
        <h2 class="title">{{title}}</h2>
        <h3 class="subtitle">{{subtitle}}</h3>
        <p class="description">{{description}}</p>
      </div>
    </div>
    <nav class="columns">
      <div class="column first"></div>
      <div class="column"></div>
      <div class="column"></div>
      <div class="column"></div>
      <div class="column"></div>
      <div class="column"></div>
    </nav>
  </div>
</template>

<script lang="ts">
import { Component, Vue } from 'vue-property-decorator';

@Component({
  components: {},
})
export default class HeadHero extends Vue {
  title: string = 'Welcome to my <space/>';

  subtitle: string = 'WHO AM I?';

  description: string = "I'm a enthusiast Software developer but first of all I consider myself as ethernal curious,</br> that loves to learn, new things everyday."
}
</script>

<style lang="scss" scoped>
.text-container {
  margin-left: 30px;
}
.container {
  border-left: 3px solid #FF0000;
  position: relative;
  max-width: 1920px;
  border-bottom: 1px solid #dfdfdf;
}
.column {
  height: 600px;
  border-right: 1px solid #dfdfdf;
}

.first{
  border: 0px;
}
.columns {
  position: relative;
  width: 100%;
  margin: 0px;
}
.absolute {
  position: absolute;
  margin-left: 30px;
  margin-top: 10%;
}
.profile {
  border: 5px solid #151616;
  border-radius: 50%;
  width: 300px;
  height: 300px;
}

.title {
  color: #3f3c3c;
  font-size: 70px;
  font-weight: bold;
  margin-bottom: 40px;
}

.subtitle {
    font-size: 22px;
    font-weight: 500;
    color: #3f3c3c;
    margin-left: 20px;
    margin-bottom: 0px;
}

p {
    margin-left: 20px;
}
</style>
