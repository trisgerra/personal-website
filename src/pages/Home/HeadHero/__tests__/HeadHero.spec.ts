import { Wrapper } from '@vue/test-utils';
import HeadHero from '../HeadHero.vue';
import createWrapper from '@/utils/CreateWrapper';

describe('Home', () => {
  it('should render as expected', () => {
    const wrapper: Wrapper<HeadHero> = createWrapper(HeadHero);
    expect(wrapper.element).toMatchSnapshot();
  });

  it('contains 6 column as background', () => {
    const wrapper: Wrapper<HeadHero> = createWrapper(HeadHero);
    const navWrapper : Wrapper<any> = wrapper.findAll('.columns').at(1);
    expect(navWrapper.findAll('.column').length).toBe(6);
  });

  it('contains profile image', () => {
    const wrapper: Wrapper<HeadHero> = createWrapper(HeadHero);
    expect(wrapper.findAll('.profile').length).toBe(1);
  });

  it('contains title', () => {
    const wrapper: Wrapper<HeadHero> = createWrapper(HeadHero);
    expect(wrapper.findAll('.title').length).toBe(1);
    expect(wrapper.find('.title').text()).toEqual('Andrea DG <website/>');
  });

  it('contains subtitle', () => {
    const wrapper: Wrapper<HeadHero> = createWrapper(HeadHero);
    expect(wrapper.findAll('.subtitle').length).toBe(1);
    expect(wrapper.find('.subtitle').text()).toEqual('WHO AM I?');
  });

  it('contains description', () => {
    const wrapper: Wrapper<HeadHero> = createWrapper(HeadHero);
    expect(wrapper.findAll('.description').length).toBe(1);
  });
});
