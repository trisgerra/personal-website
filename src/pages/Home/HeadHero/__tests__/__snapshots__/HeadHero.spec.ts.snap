// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render as expected 1`] = `
<div>
  <div
    class="container desktop is-hidden-touch"
  >
    <div
      class="absolute columns"
    >
      <div>
        <img
          alt="profile image"
          class="profile"
          src="../../../assets/profile2.jpg"
        />
      </div>
       
      <div
        class="column text-container"
      >
        <h2
          class="title"
        >
          Andrea DG &lt;website/&gt;
        </h2>
         
        <h3
          class="subtitle"
        >
          WHO AM I?
        </h3>
         
        <p
          class="description"
        >
          I'm 
          <b>
            simply
          </b>
           a Software developer with a passion for 
          <b>
            new technologies
          </b>
           and the 
          <b>
            design
          </b>
           .
        </p>
      </div>
    </div>
     
    <nav
      class="columns"
    >
      <div
        class="column first"
      />
       
      <div
        class="column"
      />
       
      <div
        class="column"
      />
       
      <div
        class="column"
      />
       
      <div
        class="column"
      />
       
      <div
        class="column"
      />
    </nav>
  </div>
   
  <div
    class="container mobile is-hidden-desktop"
  >
    <div
      class="columns"
    >
      <div
        class="column is-12-mobile"
      >
        <img
          alt="profile image"
          class="profile-mobile"
          src="../../../assets/profile2.jpg"
        />
      </div>
       
      <div
        class="column text-container"
      >
        <h2
          class="title"
        >
          Andrea DG &lt;website/&gt;
        </h2>
         
        <h3
          class="subtitle"
        >
          WHO AM I?
        </h3>
         
        <p
          class="description"
        >
          I'm 
          <b>
            simply
          </b>
           a Software developer with a passion for 
          <b>
            new technologies
          </b>
           and the 
          <b>
            design
          </b>
           .
        </p>
      </div>
    </div>
  </div>
</div>
`;
